devices.js

```diff
@@ -653,10 +653,10 @@ const devices = [
         model: 'QBKG22LM',
         vendor: 'Xiaomi',
         description: 'Aqara D1 2 gang smart wall switch (no neutral wire)',
-        supports: 'on/off, action, power measurement',
+        supports: 'on/off, action',
         fromZigbee: [
             fz.on_off_xiaomi_ignore_endpoint_4_5_6, fz.xiaomi_on_off_action, fz.legacy_QBKG03LM_QBKG12LM_click, fz.QBKG03LM_buttons,
-            fz.QBKG03LM_QBKG12LM_operation_mode, fz.generic_device_temperature,
+            fz.QBKG03LM_QBKG12LM_operation_mode,
         ],
         toZigbee: [tz.on_off, tz.xiaomi_switch_operation_mode],
```

Comment:
I see that QBKG25LM_operation_mode differs from xiaomi_switch_operation_mode, since these are both the same switches I expect the commands to be similar, does the operation mode actually work for the QBKG22LM now?

Reply:
That's right -- the QBKG22LM seems more similar to my QBKG04LMs than the QBKG25LM (e.g. no double/triple click but has hold event, doesn't turn on then off if I double click, doesn't enter pairing process if it's connected to a coordinator), which leads me to believe that hardware-wise the 1- and 2-gang D1 switches are practically the same as the 1st gen switches

Reply:
also interestingly, the QBKG25LM's depth is less than that of the QBKG22LM and QBKG04LM iirc